.gitignore

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2022 Robert Bosch GmbH
```

Comment:
Please add .gitattributes with https://github.com/eclipse-velocitas/vehicle-app-python-template/blob/main/.gitattributes same content - it solves windows endings issue (crlf vs lf)

Reply:
Done